cedar-policy/CHANGELOG.md

```diff
@@ -15,7 +15,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - `Entity::into_inner` (#685, resolving #636)
 - New `ffi` module with an improved FFI interface. This will replace the
   `frontend` module in the 4.0 release, but is available now for early adopters;
-  the `frontend` module is now deprecated. (#852)
+  the `frontend` module is now deprecated.
+  This should be considered a preview-release of `ffi`; more API breaking
+  changes are anticipated for Cedar 4.0. (#852)
+- `wasm` Cargo feature for targeting Wasm
```

Comment:
May also note that the wasm is preview release too -- more breaking changes may come once we finish up the FFI.

Reply:
Done.